tikv/raftstore/fsm_peer.go

```diff
@@ -39,6 +39,8 @@ type PeerEventObserver interface {
 	OnPeerDestroy(ctx *PeerEventContext)
 	// OnSplitRegion will be invoked when region split into new regions with corresponding peers.
 	OnSplitRegion(derived *metapb.Region, regions []*metapb.Region, peers []*PeerEventContext)
+	// OnRegionEpochChange will be invoked after conf change updated region's epoch.
+	OnRegionEpochChange(ctx *PeerEventContext, epoch *metapb.RegionEpoch)
```

Comment:
Can we name it `OnRegionConfChange`?
Because `SplitRegion` also changes the region epoch but doesn't call this function.